### PR TITLE
Add Apostrophe to Equals sign

### PR DESCRIPTION
**Pull request by pashok2398**
## Description

When deploying using Ansible module `kubernetes.core.k8s` getting error:

```
"msg": "Error loading resource_definition: could not determine a constructor for the tag
 'tag:yaml.org,2002:value'\n  in 
\"/home/paveld/go/src/github.ibm.com/IBM-Research-Galaxy/clidle/prom-operator/manifests/setup/
0alertmanagerConfigCustomResourceDefinition.yaml\", line 104, column 31"
```

```
$ ansible --version

ansible [core 2.12.10]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/paveld/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.10/site-packages/ansible
  ansible collection location = /home/paveld/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.9 (main, Dec  7 2022, 00:00:00) [GCC 12.2.1 20221121 (Red Hat 12.2.1-4)]
  jinja version = 3.0.3
  libyaml = True

```
```
$ kubectl version --output=json
{
  "clientVersion": {
    "major": "1",
    "minor": "26",
    "gitVersion": "v1.26.0",
    "gitCommit": "b46a3f887ca979b1a5d14fd39cb1af43e7e5d12d",
    "gitTreeState": "clean",
    "buildDate": "2022-12-08T19:58:30Z",
    "goVersion": "go1.19.4",
    "compiler": "gc",
    "platform": "linux/amd64"
  },
  "kustomizeVersion": "v4.5.7",
  "serverVersion": {
    "major": "1",
    "minor": "25",
    "gitVersion": "v1.25.0",
    "gitCommit": "a866cbe2e5bbaa01cfd5e969aa3e033f3282a8a2",
    "gitTreeState": "clean",
    "buildDate": "2022-09-01T23:30:43Z",
    "goVersion": "go1.19",
    "compiler": "gc",
    "platform": "linux/amd64"
  }
}
```

```
$ ansible-galaxy collection list kubernetes.core

# /home/paveld/.ansible/collections/ansible_collections
Collection      Version
--------------- -------
kubernetes.core 2.3.2  

# /usr/lib/python3.10/site-packages/ansible_collections
Collection      Version
--------------- -------
kubernetes.core 2.3.1  
```

## Type of change

_What type of changes does your code introduce to the kube-prometheus? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [x] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. Later this will be copied to the changelog file._

```
Add Apostrophe to Equals sign in manifests/setup/0alertmanagerConfigCustomResourceDefinition.yaml

```

### CODE REVIEW

**Comment by simonpasquier** [comment]
this won't work because the files in the manifests/ directory are automatically generated.
See https://github.com/prometheus-operator/prometheus-operator/issues/4955 and https://github.com/prometheus-operator/prometheus-operator/pull/4897.
TL;DR: this is a correct YAML definition and we haven't found a way to change the generated YAML output.
